app/src/main/java/org/alexcode/snake/EditNameDialog.java

```diff
@@ -51,9 +52,14 @@ public void onClick(DialogInterface dialogInterface, int i) {
                     @Override
                     public void onClick(DialogInterface dialogInterface, int i) {
```

Comment:
onClick on what exactly? is not clear enough

Reply:
On click positive button of the dialog box. Check line 51